src/graphics/camera_normal.cpp

```diff
@@ -117,22 +117,23 @@ void CameraNormal::smoothMoveCamera(float dt)
         delta = 0.0f;
     else if (delta > 1.0f)
         delta = 1.0f;
-
-    m_camera_offset += (wanted_camera_offset - m_camera_offset) * delta;
-
+    
     float delta2 = dt * 8.0f;
     if (delta2 < 0)
         delta2 = 0;
     else if (delta2 > 1)
         delta2 = 1;
 
+    m_camera_offset += (wanted_camera_offset - m_camera_offset) * delta;
```

Comment:
you edited the camera.cpp, and I'm not sure why
this line was moved, but it seems like it shouldn't matter?